app/models/country.rb

```diff
@@ -0,0 +1,4 @@
+class Country < ApplicationRecord
+  has_many :people_places, class_name: 'CountryUser', foreign_key: 'c_id'
```

Comment:
in which scenario will we use primary_key?

Reply:
if we have a column which we want to use as a primary key for that particular association other than conventional `id` column then we can pass an option `primary_key: <column_name>` .